### DISCUSSION TITLE
Proton and USB detection

### DISCUSSION THREAD

**Discussion by ThommyaVV** [scarburato/t150_driver#30] [General]
I had a problem with my game detecting my Thrustmaster T150 in Proton(it worked in native version), and I found the fix.

The fix:
Newer Proton versions may have a problem detecting USB devices in newer versions, so the fix would be to downgrade.
My fix was downgrading Proton form 6.20-GE-1 to 5.0-13, and it just worked after that.

So my goal with this post is to make you aware of this, hoping it will help. Thanks

**Comment by berarma** [comment]
I think you mean new GE versions, not just newer vanilla versions.

**Reply by ThommyaVV** [reply]
No, in my personal testing, at least proton 6.3-8 and newer did not work on one of the games I play(ETS2), so I wanted to make people aware

**Comment by scarburato** [comment]
Did you try the `hid` branch?

**Reply by ThommyaVV** [reply]
No, I haven't taken time to figure out that yet, how I've set it up works good now

**Comment by Dracks** [comment]
Hey, I just bought a steamdeck, and since I had a t150 from my ps4, I wished to connect to play some of the games of the steamdeck, and (for the moment I only tried automobilista 2, I think) I don't have it working, I tried installing this driver, and installint the oficial drivers inside the proton prefix of the game, but no luck. 

Any guide/comments on how to get it working? 
Thanks!

**Reply by albfan** [reply]
I'm unsure about steamdeck features, you have access to a terminal?

If so, what is the output on dmesg or journalctl when you connect it? Remember to use PS3 mode

**Reply by Dracks** [reply]
yes, I've got access to the terminal, Today I tried the steering wheel with oversteer, and it works, but seems the problem is inside the proton itself. (the game doesn't see the steering wheel).

The dmesg shows the steering wheel fine.

**Reply by albfan** [reply]
I would say this is then something to report on proton. If driver works, oversteer works and proton config works https://github.com/ValveSoftware/Proton/issues/4819#issuecomment-841112020, I guess it depends on proton+specific game

They talk specifically about automobilista 2
